### PR TITLE
remove all the pipelineresources bases addons

### PR DESCRIPTION
**Pull request by pradeepitm12**
this removed all the task, pipeline templates, pipeline samples
and task snippets which uses pipelineresources

aslo update tasks snippet with MINORVERSION removed

# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/operator/blob/master/CONTRIBUTING.md) for more details._

# Release Notes

### CODE REVIEW

File: cmd/openshift/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-python-task-snippet.yaml [hunk 1]
```diff
@@ -2,31 +2,31 @@
...
       workspaces:
         - name: source
           workspace: workspace
       params:
         - name: IMAGE
           value: $(params.IMAGE_NAME)
-        - name: MINOR_VERSION
-          value: "6"
+        - name: VERSION
+          value: "3.8"
```
Comment (ppitonak) [inline cmd/openshift/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-python-task-snippet.yaml:29]: why not the same version as in clustertask, i.e. `3.8-ubi8`?

File: cmd/openshift/operator/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-nodejs-task-snippet.yaml [hunk 2]
```diff
@@ -26,7 +26,7 @@ spec:
         - name: IMAGE
           value: $(params.IMAGE_NAME)
         - name: VERSION
-          value: "8"
+          value: "12"
```
Comment (ppitonak) [inline cmd/openshift/operator/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-nodejs-task-snippet.yaml:29]: why not the same version as in clustertask, i.e. `12-ubi8`? or the newest available `14-ubi8`?

File: cmd/openshift/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-python-task-snippet.yaml (hunk 1 above)
Comment (pradeepitm12) [reply cmd/openshift/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-python-task-snippet.yaml:29]: as these are snippets I thought to give a different version. 
But we can update it to the same as the clustertasks.

File: cmd/openshift/operator/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-nodejs-task-snippet.yaml [hunk 3]
```diff
@@ -26,7 +26,7 @@ spec:
         - name: IMAGE
           value: $(params.IMAGE_NAME)
         - name: VERSION
-          value: "8"
+          value: "12-ubi8"
```
Comment (ppitonak) [inline cmd/openshift/operator/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-nodejs-task-snippet.yaml:29]: why `12-ubi8` when pkg/reconciler/openshift/tektonaddon/pipelinetemplates/pipelinetemplates.go defines `14-ubi8`?

**Review by ppitonak — CHANGES_REQUESTED** [review]

File: cmd/openshift/operator/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-nodejs-task-snippet.yaml (hunk 3 above)
Comment (pradeepitm12) [reply cmd/openshift/operator/kodata/tekton-addon/0.0.1/optional/samples/tasksnippets/s2i-nodejs-task-snippet.yaml]: This I missed as it in the samples. Updated now

**Pull request merged**